commerce_coordinator/apps/frontend_app_payment/serializers.py

```diff
@@ -16,3 +18,29 @@ class GetPaymentOutputSerializer(serializers.Serializer):  # pylint: disable=abs
     Serializer for OrderFulfillView input validation.
     """
     state = serializers.CharField(allow_null=False)
+
+
+class DraftPaymentCreateViewInputSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Serializer for DraftPaymentCreateView input validation.
+    """
+    edx_lms_user_id = serializers.IntegerField(allow_null=False)
+
+
+class DraftPaymentCreateViewOutputSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Serializer for DraftPaymentCreateView input validation.
+    """
+    number = serializers.CharField(allow_null=False)
+    orderUuid = serializers.UUIDField(allow_null=False)
+    responseCode = serializers.CharField(allow_null=False)
+    state = serializers.CharField(allow_null=False)
+
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        ret = OrderedDict()
+        ret['payment_number'] = representation['number']
+        ret['order_uuid'] = representation['orderUuid']
+        ret['key_id'] = representation['responseCode']
+        ret['state'] = representation['state']
```

Comment:
These are diverted from the output mentioned in the [spec](https://github.com/edx/commerce-coordinator-private/blob/626c5fc205fa624d5673541e1dc99b1a54f5d07d/docs/api-spec/theseus-api-spec.md#output-11).

`redirect` is missing here. I was not sure of its use for us.
`payment_number` is added. 
`order_id` renamed to `order_uuid`
`state` is also added. I think this will be helpful to know about the payment status.